contrib/runners/remote_command_runner/runner.yaml

```diff
@@ -15,11 +15,11 @@
       description: Arbitrary Linux command to be executed on the remote host(s).
       type: string
     cwd:
-      default: "~/"
+      default: "/tmp"
```

Comment:
How does this change affect things?

I would imagine it's backward incompatible in some scenarios - /tmp is not necessarily always accessible to all the users. In any case, I believe it should be at least fine for our bootstrap installations.